source/_components/proximity.markdown

```diff
@@ -49,29 +49,28 @@ proximity:
 ```
 
 {% configuration %}
-proximity:
```

Comment:
Where is the proximity variable? 😆

Reply:
It's gone :D
If you look at other components this key is not present 🤷‍♂️